addons/website_slides/static/src/js/slides_course_quiz.js

```diff
@@ -190,6 +188,23 @@ odoo.define('website_slides.quiz', function (require) {
                 QWeb.render('slide.slide.quiz.validation', {'widget': this})
             );
         },
+
+        /*
+         * @private
+         * render the button to join a cours
+         */
+        _renderJoinWidget: function () {
+            var courseJoinWidget = new CourseJoinWidget(this, this.channel.channelId, true);
+            if (!this.isMember && !this.publicUser) {
+                if (this.$(".o_wslides_course_join_widget")[0]) {
```

Comment:
It will work the same way but we prefer `this.$(".o_wslides_course_join_widget").length !== 0`

It's more explicit.